src/main/java/echoServer/MockClientSocketWrapper.java

```diff
@@ -0,0 +1,50 @@
+package echoserver;
+
+import java.net.*;
+import java.io.*;
+
+public class MockClientSocketWrapper implements ClientSocketWrapperInterface {
+    private BufferedReader input;
+    private PrintWriter output;
+    private boolean socketCreated = false;
+    private boolean closeCalled = false;
```

Comment:
Do you two know about the `final` and `static` keywords in Java? Could you two do some research and see if it should be used in some of the variables throughout your project?